lib/api3/const.json

```diff
@@ -0,0 +1,3 @@
+{
+  "API3_VERSION": "3.0.1"
```

Comment:
we should use semver versioning, so I think this is `3.0.0-alpha` is more appropriate. We should change it with the release of Nightscout 0.12 to `3.0.0`

Reply:
I agree with your suggestion fully. The 3.0.1 version came from swagger online editor but we don't really need the versioning until the first official release.
So what next? Will you cancel this PR, please ? I can then prepare the version modification + some new code to review.